app/src/org/commcare/utils/MediaUtil.java

```diff
@@ -414,7 +414,7 @@ private static int getApproxScaleDownFactor(int newWidth, int originalWidth) {
         double widthScaleDownFactor =  (double)boundingWidth / originalWidth;
         // Choosing the larger of the scale down factors, so that the image still fills the entire
         // container
-        double dominantScaleDownFactor = Math.max(widthScaleDownFactor, heightScaleDownFactor);
+        double dominantScaleDownFactor = Math.min(widthScaleDownFactor, heightScaleDownFactor);

```

Comment:
Thanks for looking at it as part of this work. Though if it isn't realted to Android 10 work, would be nice to separate this out in another PR so that it's easy to revert in case of any edge cases.